README.md

```diff
@@ -72,24 +71,26 @@ docker_image_manifest_path = ""   # Path to manifest in docker image
 The default configuration of _dbtwiz_ will be installed the first time you run it, but you
 may want to adjust some settings from the get-go to fit your environment.
```

Comment:
Could add description saying that the user config only applies to linux environments.

Reply:
No longer the case.